plugins/textpattern.md

```diff
@@ -129,4 +148,8 @@ tinymce.init({
 });
 ```
 
-In this example, typing `---` will insert a horizontal rule block, but typing `(c)` will insert an inline copyright symbol. This is useful for commonly used phrases or symbols, and can be leveraged to create content templates. The last pattern is an example of this.
+Using the configuration in this example:
+* Typing `---` will insert a horizontal rule block.
```

Comment:
This should possibly say "Typing x then space or enter"?

Reply:
Fixed